cluster/images/conformance/Dockerfile.windows

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE(claudiub): We're adding busybox to the Windows image so we can use the
+# same bash script, so we can avoid maintaining multiple scripts. This building
+# method is based on kubernetes/test/images/busybox/Dockerfile_windows
```

Comment:
I'm not familiar with this flow but glad it you were able to find an example to base it on.

The bash script should be going away in a few releases and we'd just have the go runner I think, right @wilsonehusin ? At that point would we be able to remove all the busybox stuff?

Reply:
yup, the work is here: #99178

bash script is effectively gone, but the "executable path" still exists with the same name, just that it's executing a go binary now.